src/components/LoanCards/LoanCardController.vue

```diff
@@ -118,6 +118,9 @@ export default {
 		isSelectedByAnother() {
 			return this.amountLeft <= 0 && !this.isFunded;
 		},
+		isExpired() {
+			return this.loan.status === 'expired';
+		},
```

Comment:
This is the core of this PR. I'm not able to find an expired loan to test on my VM, does this look correct to you?

Reply:
Yes that should do it.